src/multivariate/multinomial.jl

```diff
@@ -140,21 +140,20 @@ function insupport(d::Multinomial, x::AbstractVector{T}) where T<:Real
     return s == ntrials(d)  # integer computation would not yield truncation errors
 end
 
-function _logpdf(d::Multinomial, x::AbstractVector{T}) where T<:Real
+function _logpdf{T<:Real}(d::Multinomial, x::AbstractVector{T})
```

Comment:
This change is incorrect, it should still be `where`

Reply:
Thanks.